snake.py

```diff
@@ -127,6 +127,7 @@ def main():
 
         if is_eating(snake, food):
             score += score_per_food
+            snake.length += 2
```

Comment:
We should use the `Snake.growth` attribute here instead of hard-coding the growth to 2.

Reply:
Also, it would be better if growth was implemented as a function in the `Snake` class and just called here on the snake object.

Reply:
but growth literally has just one task to do. Why would that require a whole function?

Reply:
It's just better organization